services/workflows-service/src/events/document-changed-webhook-caller.ts

```diff
@@ -37,14 +47,14 @@ export class EventConsumerListener {
     });
 
     const id = randomUUID();
-    const url = process.env.WEBHOOK_URL as string;
-    const environment = process.env.NODE_ENV;
-    const authSecret = process.env.WEBHOOK_SECRET as string;
+    const environment = this.configService.get<string>('NODE_ENV');
```

Comment:
env vars should be added and used from here: services/workflows-service/src/env.ts:5